### PR TITLE
update order-guidelines to EBML spec vocabulary style

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: order_guidelines.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+When one `SeekHead` Element is present, it should be the first Element in a `Segment` as it should reference the other Top-Level Elements in that `Segment` (except maybe the long list of Clusters). There is also the possibility that in some cases there are two `SeekHead` Elements like for files produced with mkvpropedit. In that case the first `SeekHead` Element is also placed first with only the position of the Top-Level Elements except the `Cluster` Elements. The second `SeekHead` Element is placed at the end and contains a lengthy list of all `Cluster` Elements (and not the other Top-Level Elements). Placing the first `SeekHead` Element in a position other than the first position of the `Segment` would make it a lot less useful So it <em>must</em> be the first Element of a `Cluster`.
```
Comment (mbunkus) [inline order_guidelines.md]: The very last part is wrong even in the prior version. A `SeekHead` is a a child of `Segment`, not of a `Cluster`.

File: order_guidelines.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+It is much easier to put the `Cues` Element at the "end" of the `Segment`, once all the `Cluster` Elements have been written, otherwise it's hard to predict beforehand the place to reserve at the beginning of the `Segment` Element. It is also not a big deal if `Cues` are at the end of the `Segment` Element, given when the user wants to seek in a Matroska stream, it is going to jump somewhere, so it can seek to the `Cues` entry, read it and then seek to the best position according to the entries of the `Cues` Element. So the `Cues` Element <em>should</em> always be written after the `Cluster` Element. However the <a href="#cues_front">`Cues` could also appear at the front</a>. In this case the size of the `Cues` is usually very small compared to the video+audio bitrate of the stream. Even on small bitrate for a full length movie, the whole `Cues` Element size represents only 0.01 to 0.1 second of download. Which is smaller than the time needed to seek on the network.
```
Comment (mbunkus) [inline order_guidelines.md]: I don't think that the word `"end"` should be put in parenthesis here. I don't even know why that was the case in the original version.

File: order_guidelines.md (hunk 1 above)
Comment (dericed) [inline order_guidelines.md]: fixed in https://github.com/Matroska-Org/matroska-specification/pull/11/commits/240384c8253f8e9b49963f001f1a4bffeeb91ef8.

**Comment by dericed** [comment]
@mbunkus please re-review

**Comment by mbunkus** [comment]
Thanks.

**Pull request merged**